src/_includes/layouts/listing-case-studies.njk

```diff
@@ -5,11 +5,13 @@ pagination:
 permalink: '/{{ locale }}/solutions/case-studies{% if pagination.pageNumber > 0 %}/page/{{ pagination.pageNumber + 1}}{% endif %}/index.html'
 ---
 
+<!-- prettier-ignore -->
```

Comment:
I started using `<!-- prettier-ignore --> for the reasons you've mentioned, I haven't been able to get VS code working nicely with nunjucks.

I do think there's some benefit to using `.html` rather than `.njks` as at least it gets us some prettier formatting of the html.

Further discussion needed I think.